### PR TITLE
Array array modifiers for contributors and identifiers MODINVSTOR-311

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: src/test/java/org/folio/rest/api/InstanceStorageTest.java [hunk 1]
```diff
@@ -941,6 +941,31 @@ public void canSearchForInstancesUsingSimilarQueryToUILookAheadSearch() {
...
+  }
+
+  @Test
+  public void arrayModifierfsIdentifiers2() {
+    canSort("identifiers = /@identifierTypeId = isbn 9781447294146", "Uprooted");
+  }
+
+  @Test
+  public void arrayModifierfsIdentifiers3() {
+    canSort("identifiers = @identifierTypeId asin", "Nod");
```
Comment (hjiebsco) [inline src/test/java/org/folio/rest/api/InstanceStorageTest.java]: No slash before @ like other test cases?

Comment (hjiebsco) [reply src/test/java/org/folio/rest/api/InstanceStorageTest.java]: Also there is no "=" before "asin". Does the new syntax support that?

Comment (adamdickmeiss) [reply src/test/java/org/folio/rest/api/InstanceStorageTest.java]: The omitted slash was a mistake.. But it shows that full text search in many cases.. (key+values in sequence gets a hit)

Comment (adamdickmeiss) [reply src/test/java/org/folio/rest/api/InstanceStorageTest.java]: It's simply a search identifiers = "@identifierTypeId asin" .. @ most likely will be ŕemoved.. ANd "identitifierTypeId asin" is left with two tokens in order .. (phrase).

**Pull request merged**